packages/monaco-graphql/src/utils.ts

```diff
@@ -112,8 +110,7 @@ export function toMarkerData(
     startColumn: diagnostic.range.start.character + 1,
     endColumn: diagnostic.range.end.character,
     message: diagnostic.message,
-    severity: 5,
-    // severity: toMonacoSeverity(diagnostic.severity),
+    severity: monaco.MarkerSeverity.Error,
```

Comment:
<img width="566" alt="image" src="https://github.com/graphql/graphiql/assets/7361780/f1dc9c76-6efa-4c21-a959-b6e696f57d80">

@acao I'm not sure if I'm right

Reply:
this is correct. it should be `monaco.MarkerSeverity.Error`, not sure how it regressed